proxycore/cluster.go

```diff
@@ -33,33 +33,65 @@ const (
 	DefaultRefreshTimeout = 5 * time.Second
 )
 
+type Event interface {
+	isEvent() // Marker method for the event interface
+}
+
 type AddEvent struct {
 	Host *Host
 }
 
+func (a AddEvent) isEvent() {
+	panic("do not call")
```

Comment:
Why this `panic` on all of these methods?

Reply:
This method should never be called because it's just a "dummy"/"marker" method because golang interfaces require at least one method.

Reply:
Ohh I see, thanks!